tgui/packages/tgui/interfaces/CyborgModuleRewriter/style.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @file
+ * @copyright 2020
+ * @author Mordent (https://github.com/mordent-goonstation)
+ * @license ISC
+ */
+
+import { block, element } from '../../util/bem';
```

Comment:
relative like this is a bit weird

Reply:
How so?

Reply:
Oh, do you mean `util` should be a top-level alias? i.e. `import { block, element } from 'util/bem';`?

Reply:
yeah, I dunno, it just seems wrong to me to do relative pathing like that, I could just be weirded out by nothing though

Reply:
It's "normal" in terms of "I've seen it lots" way, but also "weird" in a "when you want a util you think about it from the root rather than where you are" way.

I could definitely look to alias.